src/Hosting/Hosting/src/Internal/HostingApplicationDiagnostics.cs

```diff
@@ -136,10 +136,19 @@ public void RequestEnd(HttpContext httpContext, Exception exception, HostingAppl
                 StopActivity(httpContext, activity, context.HasDiagnosticListener);
             }
 
-            if (context.EventLogEnabled && exception != null)
+            if (context.EventLogEnabled)
```

Comment:
First time I'm seeing this code, so I know this is already "how it's been" but `EventLog` != `EventSource` so this name struck me as odd. I backtracked to where it's initialized and confirmed it was right but the naming seems off.

It's internal so a rename would be safe if you want to be a little more consistent... 🤷‍♂

Reply:
It's public unfortunately. That's why I never renamed it. Can do it later, I think we should make all this stuff internal anyways.